very_good_flame_game/hooks/test/pre_gen_test.dart

```diff
@@ -15,34 +16,39 @@ void main() {
     });
 
     test('populates variables', () {
-      final vars = {
-        'project_name': 'my_game',
-        'org_name': 'com.example',
-        'application_id': 'app.id',
-        'description': 'A new Flame project.',
-      };
-      when(() => context.vars).thenReturn(vars);
-
-      pre_gen.run(context);
-
-      final newVars = verify(() => context.vars = captureAny()).captured.last
-          as Map<String, dynamic>;
-
-      expect(
-        newVars,
-        equals(
-          {
-            'project_name': 'my_game',
-            'org_name': 'com.example',
-            'description': 'A new Flame project.',
-            'android_namespace': 'app.id',
-            'android_application_id': 'app.id',
-            'ios_application_id': 'app.id',
-            'macos_application_id': 'app.id',
-            'windows_application_id': 'app.id',
-          },
-        ),
-      );
+      final clock = Clock.fixed(DateTime(2020));
+      withClock(clock, () {
+        final vars = {
+          'project_name': 'my_game',
+          'org_name': 'com.example',
+          'application_id': 'app.id',
+          'description': 'A new Flame project.',
+          'current_year': clock.now().year,
```

Comment:
It should not be given as an input variable since it is not a user-defined variable (not in `brick.yaml`).

```suggestion
```

Reply:
Thanks for pointing that out! I’ve removed the unnecessary line as per your suggestion. It’s all cleaned up now.